cmd/infracost/testdata/breakdown_arm_template_config_file/standard_disk.json

```diff
@@ -0,0 +1,30 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "metadata": {
+      "_generator": {
+        "name": "bicep",
```

Comment:
not very important, but the name here could be changed to bicep in all the json files

Reply:
I think this is here, because this file was directly generated from a bicep file. So it's not really the name of the resource

Reply:
do we want to let any traces of bicep in our PR?

Reply:
hmm shouldn't be an issue, but i'll remove it